style.css

```diff
@@ -349,3 +363,250 @@ input.form-control::placeholder {
+
+  .logo a {
+    padding-left: 30px;
+    margin-left: 55px;
+  }
+
+  .menu-content ul {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    list-style: none;
+    padding-right: 45px;
+  }
+
+  .menu-content li {
+    padding: 1rem;
+  }
+
+  .menu-content a {
+    text-decoration: none;
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .menu-content {
+    display: flex;
+  }
+
+  .intro {
+    background-image: url(./img/header_bg-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: #fff;
+    background-position: center;
+  }
+
+  .intro h2 {
+    color: #091e42;
+    letter-spacing: 0.37px;
+    padding-left: 120px;
+    margin: 180px 0 15px 0;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .intro p {
+    margin: 0 35px 0 120px;
+  }
+
+  .connect {
+    padding-left: 122px;
+    margin-top: -30px;
+  }
+
+  .socials {
+    display: flex;
+    justify-content: space-between;
+    width: 13%;
+    padding-top: 15px;
+    margin-bottom: 100px;
+    margin-left: 123px;
+  }
+
+  /* Grid */
+  .grid-container {
+    display: grid;
+    grid-template-columns: auto;
+    width: 100%;
+    padding: 5%;
+  }
+
+  .cards-img {
+    width: 8%;
+    height: 290px;
+  }
+
+  .tonic-img {
+    display: none;
+  }
+
+  .desktop-img {
+    display: inline-block;
+    width: 500px;
+    height: 448px;
+  }
+
+  .work-list {
+    display: none;
+  }
+
+  .desktop-lang {
+    display: flex;
+    margin-bottom: 69px;
+  }
+
+  .desktop-lang img {
+    margin-left: 10px;
+  }
+
+  .content-wrap {
+    display: flex;
+    flex-direction: column;
+    height: 290px;
+    margin-left: 65%;
+    margin-top: -20%;
+  }
+
+  .content-wrap2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 15%;
+    margin-right: 0;
+  }
+
+  .cards {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    background: #fff;
+    border: 2px solid #dfe1e6;
+    border-radius: 16px;
+    width: 80%;
+    height: 483px;
+    margin-left: 10%;
+  }
+
+  .h2 {
+    font-weight: 700;
+    line-height: 52px;
+    color: #091e42;
+    margin-top: -80px;
+    margin-left: -20%;
+  }
+
+  .para {
+    color: #344563;
+    line-height: 24px;
+    font-size: 14px;
+    font-weight: 400;
+    margin-bottom: 15px;
+    margin-left: -20%;
+    margin-right: 10px;
+  }
+
+  .top {
+    padding-right: 10px;
+    margin-top: -10px;
+    margin-left: -20%;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .projects ul {
+    padding-right: 99px;
+    margin-top: -3px;
+    margin-left: -60px;
+  }
+
+  .btn {
+    margin-left: -55px;
+  }
+
+  .text-2 {
+    display: none;
+  }
+
+  .container-section2 {
+    display: grid;
+    grid-template-columns: repeat(8, 1fr);
+    padding-top: 142px;
+    margin: 100px auto;
+    width: 100%;
+  }
+
+  .intro2 {
+    margin-left: 80px;
+    width: 400px;
+    margin-top: -100px;
+  }
```

Comment:
- [ ] Please fix the width of these sections to have `40%` on the left and `60%` on the right in order to match the design as previously asked by the first reviewer. 

** Your design **
![image](https://user-images.githubusercontent.com/50941074/160216769-a1d47509-a9c1-4d20-8bf4-fcfd1a36ed80.png)

** The Figma Design **
![image](https://user-images.githubusercontent.com/50941074/160216804-addbe9ca-70ae-41e3-8249-1dad84f9a68b.png)